src/lib/arrays.js

```diff
@@ -0,0 +1,100 @@
+const getNthElement = (index, array) => {
+  index %= array.length;
+  return array[index];
+};
+
+const arrayToCSVString = array => {
+  return array.join();
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  const newArray = [...array, element];
+  return newArray;
+};
+
+const removeNthElement = (index = 0, array) => {
+  array.splice(index, 1);
+  return array;
+};
+
+const numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings
+    .join()
+    .toUpperCase()
+    .split(',');
+};
+
+const reverseWordsInArray = strings => {
+  return strings
+    .toString()
+    .split('')
+    .reverse()
+    .join('')
+    .split(',')
+    .reverse();
+};
+
+const onlyEven = numbers => {
+  function even(n) {
+    return !(n % 2);
+  }
+  return numbers.filter(even);
+};
+
+const removeNthElement2 = (index, array) => {
+  const firstArray = array.slice(0, index);
+  const secondArray = array.slice(index + 1, array[-1]);
+  return firstArray.concat(secondArray);
+};
+
+const elementsStartingWithAVowel = strings => {
+  const vowelWords = strings.filter(word => /^[aeiou]/i.test(word));
+  return vowelWords;
```

Comment:
No need to create the extra variable to just return it on the next line, can just return the result of the filter method directly:
```
  return strings.filter(word => /^[aeiou]/i.test(word));
```